elements/pfe-styles/src/pfe-base.scss

```diff
@@ -82,22 +82,19 @@ th {
 // }
 
 body {
-  font-family: pfe-var(font-family);
+  @include pfe-text-base;
   font-size:   var( --pf-c--FontSize, pfe-var(content--FontSize));
-  font-weight: pfe-var(font-weight--light);
-  line-height: pfe-var(line-height);
   color: pfe-var(text);
   text-align: left;
   margin: 0;
 }
 
 // HTML TYPOGRAPHY
 .pfe-c-content {
-
   // HEADINGS
   @each $heading-size in (h1, h2, h3, h4, h5, h6) {
     #{$heading-size}  {
-        @extend %#{$heading-size}; 
+        @include pfe-content-heading (#{$heading-size}); 
```

Comment:
Yes, this is where we'd see the efficiency in the rendered result of using the extend for the pfe-text-base instead of a mixin for that part.